.github/workflows/pypi-publish.yml

```diff
@@ -27,4 +27,4 @@ jobs:
       run: |
         python3.7 -m pip install --upgrade pip
         python3.7 -m pip install virtualenv pip-tools
-        pip install ${{ secrets.PACKAGE }}
+        python3.7 -m pip install ${{ secrets.PACKAGE }}
```

Comment:
What's the point of this? The name suggests it's installing dependencies (and is the same name as a previous step), but dependencies of what? And it's the last step in this workflow, so it seems like it's an installation for dependencies that doesn't go anywhere.

Reply:
This step goes and installs the package are its uploaded from pypi. I started doing this after the aml team would frequently publish wheels which would thrn fail to install due to depedancy or packaging issues.

Reply:
You can do that locally once you build the wheels if you just want a sanity check. But I would do this **before** uploading to PyPI, not after.